Issue 9 - Modelo de template para README.md

Eduardojvr opened the pull request:
Adicionado template inicial de README que os jogos deverão seguir.

Eduardojvr committed:
Adicionando pasta de imagens temporária

Co-Authored-By: Eduardojvr <eduardojvr10@gmail.com>
Co-Authored-By: lugope <at97_@hotmail.com>

Eduardojvr committed:
Adicionando template inicial

issue #9

Co-Authored-By: Eduardojvr <eduardojvr10@gmail.com>
Co-Authored-By: lugope <lucasgpereira.10@gmail.com>

Eduardojvr committed:
Atualizando template

issue #9

Co-Authored-By: Eduardojvr <eduardojvr10@gmail.com>
Co-Authored-By: lugope <lucasgpereira.10@gmail.com>

Eduardojvr committed:
Modificando readme para a realização correta do parser

issue #9

ednunes commented:
Sugestões de mudança observadas:

- Trocar o nome do arquivo de README.md para unbgames.md

- Trocar os títulos:
    "Development" para "Developers" (Linha 30)
    "Art" para "Designers" (Linha 35)
    "Music" para "Musicians" (Linha 40)
 
- Em Gallery substituir por subtópicos:
    ## Cover Image
        Link da imagem de Capa
    ## Carousel Gallery 
        Link das imagens que estarão no carousel da aplicação
    ## Image Gallery
        Link das imagens que estarão na galeria de imagens da aplicação

Nos links das imagens colocar somente o link ao invés da tag html.

- Falta criar o subtópico da galeria de músicas

Eduardojvr committed:
Atualização de template

Atualização de template de acordo com alguns feedbacks. @eduardonunes2525  modificamos o template com algumas das sugestões dadas no pull  request, como temos muita coisa para fazer no projeto, achamos que outras melhorias no template e no script de parser poderão serem feitas no decorrer do projeto.
